### PR TITLE
test/e2e_profile

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?

https://trello.com/c/zU78cvJZ/210-end-to-end-test-for-profile-page

> If this is an issue, do we have steps to reproduce?

N/A

### Intent

> What changes are introduced by this PR that correspond to the above card?

Cypress tests added covering the profile page.

> Would this PR benefit from screenshots?

### Considerations

> Is there any additional information that would help when reviewing this PR?

Uncovered a potential issue regarding testing for UTC around money, but the date_fns library does not support UTC in this version.

> Are there any steps required when merging/deploying this PR?

N/A

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/routes/__tests__/money.spec.js [hunk 1]
```diff
@@ -339,8 +339,8 @@ describe('Prisoner Money', () => {
           expect(spy).toHaveBeenCalledWith(
             testUser,
             'spends',
-            new Date('2021-01-01T00:00:00.000Z'),
-            new Date('2021-01-31T23:59:59.999Z'),
+            new Date('2021-01-01T00:00:00.000'),
+            new Date('2021-01-31T23:59:59.999'),
```
Comment (jonhindle) [inline server/routes/__tests__/money.spec.js:343]: Removing UTC from these tests. The code they work on, only works with local (date_fns). Worth raising a ticket to investigate using the UTC instead.

File: cypress/features/profile.feature [hunk 2]
```diff
@@ -8,6 +8,113 @@ Feature: Profile
...
+    Then I am shown the following closed sensitive card
+    | dataTest  | moneyPrivate                    |
+    | title     | Private                         |
+    | content   | Private account current balance |
+    | sensitive | £5.00                           |
+    Then the "moneyPrivate" sensitive card is closed
+    When I click on the "moneyPrivate" card
+    Then the "moneyPrivate" sensitive card is open
+    Then I am shown the following closed sensitive card
+    | dataTest  | moneySavings                    |
```
Comment (andrewrlee) [inline cypress/features/profile.feature:92]: meh.. the levels of abstraction in these tests seem quite varied.

could call this attribute `card | moneySavings` or something? 

`dataTest` seems to be very specific, and referring to this as a card would raise the level of abstraction out of

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**